lib/tests/crud.test.js

```diff
@@ -32,6 +32,7 @@ jest.mock("@aws-sdk/client-lambda", () => {
                             some: "form",
                           },
                           publishingStatus: true,
+                          securityAttribute: "Unclassified",
```

Comment:
Is there any reason this has an uppercase "U"? As more values are added, it might be easier if everything was lowercase.

Reply:
[Security attribute list](https://github.com/cds-snc/platform-forms-client/issues/701#issuecomment-1098166716)

Reply:
[Security attribute list](https://github.com/cds-snc/platform-forms-client/issues/701#issuecomment-1098166716)